websocket-ballerina/tests/websocket_server_exceptions.bal

```diff
@@ -1,61 +1,62 @@
-//// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Need to think of a way to rewrite this test once this [issue](https://github.com/ballerina-platform/ballerina-standard-library/issues/828) gets fixed. Kept the test commented for reference.